# dpad
bool up
bool down
bool left
bool right
---
string response